structured/sql_api.proto

```diff
@@ -93,6 +93,6 @@ message SQLRequest {
 
 message SQLResponse {
   optional SQLResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
-  // The result set.
+  // The result set for the last Cmd in the request.
```

Comment:
This is what I found when I looked at the postgres implementation. I suppose each query can have its own set of columns and it would be difficult to disambiguate between them in the response.